auth_proxy/fixtures.yml

```diff
@@ -2,19 +2,24 @@
 class: !!python/name:auth_proxy.models.user.User
 args:
   patient_id: smart-1288992
-  username: demo-smart-user
+  username: David Jones
```

Comment:
David really wanted to log in with a person's name, not a username. Instead of reworking the login behavior, this was easier.